doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,280 @@
+  available cmorizers). 
+
+It is recommended to use an existing custom table as a template, to edit the content and save it as
+``<short_name>.dat``.
+
+2. Edit your configuration file
+=====================
+
+Make sure that beside the paths to the model simulations and observations, also the path to 
+raw observational data to be cmorized (``RAWOBS``) is present in your configuration file.
+
+3. Store your dataset in the right place
+========================================
+
+The folder ``RAWOBS`` needs the subdirectories ``Tier1``, ``Tier2`` and ``Tier3``. 
+The different tiers describe the different levels of restrictions for downloading 
+and using the observations. The unformatted (raw) observations should then be 
+stored then in the appropriate of these three folders. 
+
+4. Create a cmorizer for the dataset
+========================================================
+
+There are many cmorizing scripts available in `/esmvaltool/cmorizers/obs/
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/>`_
+where solutions to many kinds of format issues with observational data are
+addressed. Most of these scripts are written in NCL at the moment, but more 
+and more examples for Python-based cmorizing scripts become available.
+
+How much cmorizing an observational data set needs is strongly dependent on
+the original NetCDF file and how close the original formatting already is to
+the strict CMOR standard. 
+
+In the following two subsections two cmorizing scripts, one written in python and
+one written in NCL, are explained in more detail.
+
+4.1 Cmorizer script written in python
+*************************************
+
+Find here an example of a cmorizing script, written for the ``MTE`` dataset
+that is available at the MPI for Biogeochemistry in Jena: `cmorize_obs_mte.py
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/cmorize_obs_mte.py>`_.
+
+All the necessary information about the dataset to write the filename correctly, 
+and which variable is of interest, is stored in a seperate configuration file: `MTE.yml
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/cmor_config/MTE.yml>`_.
+
+The first part of this configuration file defines the filename of the raw
+observations file, the second part defines the common global attributes for 
+the cmorizer output, e.g. information that is needed to piece together the 
+final observations file name in the correct structure (see Section 6). The 
+third part defines the variables that are supposed to be cmorized.
+
+The actual cmorizing script ``cmorize_obs_mte.py`` consists of a header with
+information on where and how to download the data, noting the last access of the
+data webpage, and who has written the cmorizing script. 
+
+After the header there is a section where different procedures and functions
```

Comment:
I would replace this and the next paragraph by something like this (it's not important to explain every detail of this specific CMORizer but to focus on the necessary elements):

```rst
The main body of the CMORizer script must contain a function called

.. code-block:: python

   def cmorization(in_dir, out_dir, cfg):

with this exact call signature. Here, ``in_dir`` corresponds to the input directory of the raw files,
``out_dir`` to the output directory of final reformatted data set and ``cfg`` to the configuration
dictionary given by the  ``.yml`` configuration file. This function needs to return ``None``. All the
work, i.e. loading of the raw files, processing them and saving the final output, has to be
performed inside its body. To simplify this process, ESMValTool provides a set of predefined utilities_,
which can be imported into your CMORizer by

.. code-block:: python

   from . import utilities as utils

Apart from a function to easily save data, this module contains different kinds of small fixes
to the data attributes, coordinates, and metadata which are necessary for the data field to be
CMOR-compliant.

Note that this specific CMORizer script contains several subroutines in order to make the code more clearer
and readable (we strongly recommend to follow that code style). For example, the function
``_get_filepath`` converts the raw filepath to the correct one and the function ``_extract_variable``
extracts and saves a single variable from the raw data.

.. _utilities: https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/utilities.py
```

Reply:
Done!